frontend/src/pages/documentation/static-dataflow-analysis.mdx

```diff
@@ -0,0 +1,156 @@
+import JavaKotlinCodeBlock from "../../components/JavaKotlinCodeBlock";
+import Examples from "../../components/examples";
+
+# Static dataflow analysis
+
+With the `jacodb-analysis` module, one can perform static dataflow analysis based on the three-address code intermediate representation.
+It implements the IFDS solver, described in the <a href="https://dx.doi.org/10.1145/199448.199462">Reps, Horwitz, and Sagiv (1995)</a> paper.
+IFDS is the acronym for the class of the _interprocedural_, _finite_, _distributive_, _subset_ problems.
+
+The `jacodb-analysis` module provides you with several ready-to-use analyses along with an API to build your own analyses.
+
+Current implementation splits code into `Units`, so that the IFDS framework can analyse them concurrently.
+Information is shared between the units via `Summaries`, but the lifecycle of each unit is controlled
+separately. This makes the implementation highly scalable and still precise.
+
+## Basic usage
+
+You can perform analysis right from your code or via command-line interface.
+
+### Calling from your code
+
+The analysis entry point is the `runAnalysis` method called from `AnalysisMain`.
```

Comment:
I believe that "called from `AnalysisMain`" is misleading here. The method can be called from anywhere. `AnalysisMain` is just a declaration place. I think `called` should be replaced with `declared` in this sentence.